lib/ddtrace/span.rb

```diff
@@ -21,6 +14,8 @@ module Datadog
   #
   # rubocop:disable Metrics/ClassLength
   class Span
```

Comment:
Hmm. I wonder if for compatibility, we should rename `SpanOperation` => `Span` and `Span` => `CompletedSpan` or something like that?

It looks to me like, after all these changes, `Span` is essentially unrecognizable, and `SpanOperation` was what we always called span, so perhaps recovering that name would be helpful for others as well?